packages/core/src/components/StepList/StepList.scss

```diff
@@ -0,0 +1,413 @@
+    height: 1px;
+    left: -$step-list-padding-left;
+    position: absolute;
+    right: -$step-list-padding-h;
+  }
+
+  @media (min-width: $step-list-breakpoint) {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+.ds-c-step--current,
+.ds-c-step--completed {
+  padding-bottom: $step-margin;
+
+  &::before {
+    border: 0 none;
+    color: $color-white;
+    line-height: $step-list-number-size-mobile;
+
+    @media (min-width: $step-list-breakpoint) {
+      line-height: $step-list-number-size;
+    }
+  }
+}
+
+.ds-c-step--current {
+  &::before {
+    background: $current-step-color;
+  }
+}
+
+.ds-c-step--completed {
+  .ds-c-step__substeps {
+    color: $color-base;
+  }
+
+  &::before {
+    background: $current-step-color;
+  }
+}
+
+.ds-c-step__content {
+  @media (min-width: $step-list-breakpoint) {
+    padding-right: $spacer-2;
+  }
+}
+
+.ds-c-step__title {
+  font-size: $h5-font-size;
+  line-height: $step-button-height;
+  margin: 0;
+
+  @media (min-width: $step-list-breakpoint) {
+    font-size: $h3-font-size;
+  }
+}
+
+.ds-c-step__actions {
+  font-size: $small-font-size;
+
+  .ds-c-step__content--with-content + & {
+    margin-top: $spacer-2;
+
+    @media (min-width: $step-list-breakpoint) {
+      margin-top: 0;
+    }
+  }
+
+  .ds-c-step--completed & a {
+    font-weight: $font-normal;
+  }
+
+  @media (min-width: $step-list-breakpoint) {
+    font-size: $base-font-size;
+    text-align: right;
+  }
+}
+
+.ds-c-step__completed-text {
+  display: inline-block;
+  line-height: $step-button-height - 12px;
+  margin-right: $spacer-2;
+  padding-top: 6px;
+
+  &::before {
+    background-image: url('#{$image-path}/checkmark-green.svg');
+    background-size: contain;
+    box-sizing: border-box;
+    content: '';
+    display: inline-block;
+    height: 12px;
+    margin-right: $spacer-1;
+    width: 16px;
+  }
+
+  @media (min-width: $step-list-breakpoint) {
+    display: block;
+    margin-right: 0;
+  }
+}
+
+.ds-c-step__description {
+  font-size: $small-font-size;
+  margin-bottom: $spacer-1;
+
+  .ds-c-step--current &,
+  .ds-c-step--completed & {
+    margin-top: $spacer-1;
+  }
+
+  @media (min-width: $step-list-breakpoint) {
+    font-size: $base-font-size;
+  }
+}
+
+.ds-c-step__substeps {
+  list-style-type: none;
+  margin: 0;
+  padding-left: $spacer-5;
+}
+
+.ds-c-substep {
+  display: block;
+  font-size: $small-font-size;
+
+  &::before {
+    content: '\2022';
+    left: -$spacer-3;
+    position: relative;
+  }
+}
+
+.ds-c-substep__title {
+  display: inline-block;
+  margin-right: $spacer-1;
+}
+
+.ds-c-substep__edit {
+  @media (min-width: $step-list-breakpoint) {
+    position: absolute;
+    right: 0;
+  }
+}
+
+/*
+Start, Resume, and Edit links
+
+A user interacts with the steps through "Start", "Resume", and "Edit"
+links.
+
+Visually "Start" and "Resume" are buttons, but all three are implemented
+with `<a>` tags. In the React component, the links' `href` properties are
```

Comment:
✍️ We could cut this first sentence. These are details people can inspect if they're really curious, but I think for the majority of people they're not going to care much about what HTML elements we use.